### ISSUE TITLE
Feature to update object/file information in Merritt

### ISSUE THREAD

**Issue by terrywbrady** [CDLUC3/mrt-doc#1720]
- Correct mime types (from file identification process)
- Augment file information with data beyond mime type (such as puid)
```
                Status: PARTIAL 
                  MrSID Image Format (Multi-resolution Seamless Image Database) (application/octet-stream): ["Droid"]
                    puid: fmt/392
```
- Enable the ability to repair/change Nuxeo file paths that are incorrect

**Comment by elopatin-uc3** [comment]
See also #1013

**Comment by terrywbrady** [comment]
Closing.  Included in #1775
